WebMvc/Models/CatalogItem.cs

```diff
@@ -7,6 +7,9 @@ namespace WebMvc.Models
 {
     public class CatalogItem
     {
+        internal object Id;
```

Comment:
hey Sunny, what this is used for? I haven't seen this in the code for this assignment before. Same for internal string Name on line 11. Can you explain?

Reply:
The Id part made a red underline and to fix this problem, it suggested me to create an object. I'll just remove this for now and see if it still makes red underline after merging all of the works.